Lib/idlelib/idle_test/test_configdialog.py

```diff
@@ -26,7 +26,7 @@
 # For testing, record args in a list for comparison with expected.
 changes = []
```

Comment:
Explicit wrote down `root` and `configure`, too.
e.g. `root = None`, `configure = None` and add two blank line between them and class TestDialog